crates/net/eth-wire/src/types/status.rs

```diff
@@ -323,7 +325,7 @@ mod tests {
 
```

Comment:
The original chainspec in this test should be using `.chain(Chain::from_id(1337))`